stix_shifter_modules/rhacs/configuration/config.json

```diff
@@ -1,8 +1,7 @@
 {
     "connection": {
         "type": {
-            "displayName": "Red Hat Advanced Cluster Security for Kubernetes (StackRox)",
-            "group": "rhacs"
+            "displayName": "Red Hat Advanced Cluster Security for Kubernetes (StackRox)"
```

Comment:
Can you keep `"group": "rhacs"` in the config? It's used for identifying connectors for the same group of products that could share connection information.